writeups/base-drafts.md

```diff
@@ -0,0 +1,312 @@
+<!--
+(12) Describe how the document meets any required formal review criteria, such
+as the MIB Doctor, YANG Doctor, media type, and URI type reviews.
+-->
+
+No formal review requirements are applicable to this document set.
+
+
+## 13. Split References
+<!--
+(13) Have all references within this document been identified as either
+normative or informative?
+-->
+
+All references within this document set have been identified as either normative
+or informative.
+
+
+## 14. Normative References
+<!--
+(14) Are there normative references to documents that are not ready for
+advancement or are otherwise in an unclear state? If such normative references
+exist, what is the plan for their completion?
+-->
+
+The document set contains the following normative references to I-Ds:
+
+* draft-ietf-httpbis-cache
+* draft-ietf-httpbis-semantics
+
+All of these are on track for timely publication in their respective WGs.
+
+
+## 15. Downward References
+<!--
+(15) Are there downward normative references references (see RFC 3967)? If so,
+list these downward references to support the Area Director in the Last Call
+procedure.
+-->
+
+There are no normative downward references in the document set.
+
+
+## 16. RFC Status Changes
+<!--
+(16) Will publication of this document change the status of any existing RFCs?
+Are those RFCs listed on the title page header, listed in the abstract, and
+discussed in the introduction? If the RFCs are not listed in the Abstract and
+Introduction, explain why, and point to the part of the document where the
+relationship of this document to the other RFCs is discussed. If this
+information is not in the document, explain why the WG considers it unnecessary.
+-->
+
+Publication of this document set will not change the status of any existing
+RFCs.
+
+
+## 17. IANA Considerations Review
+<!--
+(17) Describe the Document Shepherd's review of the IANA considerations section,
+especially with regard to its consistency with the body of the document. Confirm
+that all protocol extensions that the document makes are associated with the
+appropriate reservations in IANA registries. Confirm that any referenced IANA
+registries have been clearly identified. Confirm that newly created IANA
+registries include a detailed specification of the initial contents for the
+registry, that allocations procedures for future registrations are defined, and
+a reasonable name for the new registry has been suggested (see RFC 8126).
+-->
+
+The IANA considerations of the document set have been reviewed and no issues
+were identified.
+
+
+## 18. New "Expert Review" Registries
+<!--
+(18) List any new IANA registries that require Expert Review for future
+allocations. Provide any public guidance that the IESG would find useful in
+selecting the IANA Experts for these new registries.
+-->
+
+The document set defines several IANA registries that offer “Provisional
+Registrations” and require Expert review. The IESG should select subject matter
```

Comment:
```suggestion
The document set defines several IANA registries that offer "Provisional
Registrations" and require Expert review. The IESG should select subject matter
```
I question the use of smart quotes.  Same in @LPardue's suggestion.

Reply:
Will fix.